### PR TITLE
Refactor Home component

### LINKED ISSUE #124: Home/index.js component should be splited to ease contributions
I propose to split the Home component in three components:

- `Activity.js` with all the code concerning the `getActivity` function
- `ActivityStream.js` with the `main-chat` component part
- and `index.js` for whatever remains

What do you think ?

### PR DESCRIPTION
**Pull request by jrmi**
Split Home component in 4 files. 

Fix #124. 

Sorry I've deleted previous PR by mistake. This one only split the component and doesn't fix other bugs.

### CODE REVIEW

File: client/src/components/Home/ActivityStream.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+import styles from './styles.module.scss'
+
+const ActivityStream = ({ activities, openModal }) => {
+  const [focusChat, setFocusChat] = React.useState(false);
+  const [scrolledToBottom, setScrolledToBottom] = React.useState(true);
+  const messageStream = React.useRef(null);
+  const activitiesList = React.useRef(null);
+
+  React.useEffect(() => {
```
Comment (alanfriedman) [inline client/src/components/Home/ActivityStream.js:16]: Looks like there are still some changes here outside of reorganizing the modules. Possible to apply these changes in a separate PR?

Comment (jrmi) [reply client/src/components/Home/ActivityStream.js:16]: You mean migrating to functional component ?

Comment (jrmi) [reply client/src/components/Home/ActivityStream.js:16]: See #145

Comment (jrmi) [reply client/src/components/Home/ActivityStream.js]: Rebased after #145 merge.

**Pull request merged**